src/isar/robot/robot_status.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+from threading import Event, Thread
+from typing import Optional
+from isar.config.settings import settings
+from isar.models.communication.queues.queues import Queues
+from isar.robot.robot_queue_utils import trigger_event
+from isar.services.utilities.threaded_request import (
+    ThreadedRequest,
+)
+from robot_interface.models.mission.status import RobotStatus
+from robot_interface.robot_interface import RobotInterface
+
+
+class RobotStatusThread(Thread):
```

Comment:
Should this be running at all times or be requested updates on for the state machine?

Reply:
Yes, I can update it

Reply:
Seems like the BlockedProtectiveStop re-added the robot status thread in Idle in a rebase. I will remove it again and only use this service.